### PR TITLE
Register pratham-gupta.is-a.dev

### PR DESCRIPTION
**Pull request by prathamguptaji**
Register pratham-gupta.is-a.dev with CNAME record pointing to 69.30.249.53.

### CODE REVIEW

File: domains/pratham-gupta.json
Comment (CuteDog5695) [inline domains/pratham-gupta.json:9]: wrong record, change CNAME to A

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/pratham-gupta.json
Comment (shygamer123) [reply domains/pratham-gupta.json:9]: okay

**Review by iostpa — APPROVED** [review]
seems good to me now, wait for a maintainer to merge it then you have the domain
Edit: holy shit i need to pay more attention, accidentally forgot about the [] when it comes to A records

**Review by iostpa — CHANGES_REQUESTED** [review]
wait nvm add [] to the numbers, sorry. forgot about that

**Comment by SukiPlayz** [comment]
@shygamer123  your json file should look like this:
```json
{
    "owner": {
            "username": "shygamer123",
            "email": "gpratham522@gmail.com",
            "discord": "840244590870003762"
        },
        "record": {
            "A": ["69.30.249.53"]
        }
    }
```

update to that^^

**Review by SukiPlayz — APPROVED** [review]

**Review by iostpa — APPROVED** [review]
looks good to me now

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**
